data/campaigns/Under_the_Burning_Suns/_main.cfg

```diff
@@ -148,10 +148,6 @@
             name=_"And special thanks to everyone else who I forgot to mention."
         [/entry]
     [/about]
-
-    [load_resource]
-        id=utbs_ability_events
-    [/load_resource]
```

Comment:
The whole changes feels like a regression to me. I'm not blocking the PR for that, but the whole feature seems to be taking code that (of necessity) was split into separate functions, and putting that code into blocks that are doing other things too.

Reply:
> putting that code into blocks that are doing other things too.

Hmm, but are they not actually doing the _same_ things? Each ability (or special) block defines everything the ability does. If it is doing one of the few lucky things already "hard coded" into the C++ engine then no custom event handlers are needed, but for everything else they are integral.

Reply:
@CelticMinstrel and @Dalas121 what are your opinions on this?

Reply:
@CelticMinstrel and @Dalas121 any thoughts?

Reply:
I'm probably not the right person to ask about WML style. As long as the refactored abilities function the same as the old ones, the player experience shouldn't be harmed.

Reply:
Alright, so then this just needs an opinion from @CelticMinstrel